projects/bp-strapi/src/api/contact/controllers/contact.js

```diff
@@ -0,0 +1,29 @@
+"use strict";
+
+module.exports = {
+  contact: async (ctx) => {
+    const body = ctx.request.body;
+    const adresses = require("../config/adresses.json");
+    const sendTo = adresses[body.recipient];
+    const sender_name = body.sender_name;
+    const reply_email = body.email;
+    const subject = body.subject;
+    const message = body.message;
```

Comment:
You could use object destructuring here:
```suggestion
    const { sender_name, email: reply_email, subject, message } = body;
```